assets/app/scripts/controllers/overview.js

```diff
@@ -289,26 +291,26 @@ angular.module('openshiftConsole')
         return;
       }
       // Make sure we have a build output
-      if (!build.parameters.output.to) {
+      if (!build.spec.output.to) {
         return;
       }
-      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
-        var trigger = deploymentConfig.triggers[i];
+      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
+        var trigger = deploymentConfig.spec.triggers[i];
         if (trigger.type === "ImageChange") {
           var triggerImage = trigger.imageChangeParams.from.name;
```

Comment:
I think this comparison should reuse the `imageObjectRef` filter.

1. Update `imageObjectRef` to take an optional second arg as a namespace to use as the default if unspecified (rather than "", which is what it defaults to today), which will resolve a TODO in that function.
2. Do something like this:
```
var buildOutputImage = imageObjectRefFilter(build.spec.output.to, build.metadata.namespace);
var deploymentTriggerImage = imageObjectRefFilter(trigger.imageChangeParams.from, deploymentConfig.metadata.namespace);
if (buildOutputImage != deploymentTriggerImage) {
  continue;
}
```